core/templates/dev/head/domain/feedback_message/FeedbackMessageSummaryObjectFactory.ts

```diff
@@ -17,12 +17,24 @@
    message domain objects.
  */
 
-var oppia = require('AppInit.ts').module;
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+
+export class FeedbackMessageSummary {
+  messageId: number;
+  text: string;
+  updatedStatus: any;
```

Comment:
You should be able to figure out that this (and other fields) are strings (cf @DubeySandeep's comment below). Please fix throughout. This PR is overusing "any".

I think, in order to prevent this in the future, each field marked "any" should have a TODO above it explaining why it can't be resolved to a specific type. These TODOs should point to an issue which accumulates them explicitly so that we can keep track of them and fix them.

/cc @kevinlee12

Reply:
Ok done!